games.yaml

```diff
@@ -643,6 +697,15 @@
+LP2tzYfrOOB+c/8Z/4fOf9ZAFgAWAASLwAqfzD/mf/M/yPnv20g1I5bA6CdiNLA0C4A0gDsDYh1Ioyuv9X/2vPWuLf6o3Ui1OJdm4Bav2jt9tK78HothMx/5j/z/8j5zwLAAsACkHgBYP4b2sH8Z/4fNv/HTMDeiad1iNbh1gB5DTypnHagjxoI2vqsE8EaB+v4s/bX2k5tItDG39pvbTxGjXfmP/Of+Z9i/rMAsACwACReAJj/He1h/jP/N5//tg73DjjtwJIWntoCYB2I1glmXfik9nn1X7JTu65d4Gt+0C70UntWHX/S9d7+W/tdO9867lf3P/Of+c/8Hzr/1w4ACwALAAsAAoD5z/xn/g8UANaO1wxIDdIGXpp42g5qHaMNhHUBkNpp7b+0AFr9L/lZape235Jd68S0LgTS+Gu1b+2/dQHQxkm7ALbOA+Y/85/5f+T8ZwFgAWABSLwAMP8VcWD+M/8Pnf9tgdBOsN4FRypnnYDaBUjbHuGdKVoAAAczSURBVOuAqw0A7cCTFiLrwLT2u3UC1PzmtfBr46OdUFp/aBOOdQHQtqthwptg/jP/mf9Hz38WABYAFoDECwDzn/nP/M87/20DzzoBrAuCdgC0OsC6AEiB6F14Wxe+0n7rBLAugNYJKS3E1gXXmoi0/W+NT+vCJ/Vf6+fWcS/NA+Y/85/5n2L+swCwALAAJF4AmP/Mf+Z/Y3y2n/+thmsTQDJYc4B1YEr1aq9rAy/5pdVOK9oFoLefWv+3DkzrgltbAFsnonVB80KbiHrrb00A1vHH/Gf+a/zB/H/vt6j5jwAQ+scCwAIwglUWAOZ/27xo9bcXzH/mv6Z+hV90FWgHUG9DtfbLAGoHcGs/eydWa6CsC7jWvnYBl/y/av+9JkqU/2eNP+u8nGWH+R87/pj/sf6fNf4u68QcHQDrAGQBYAFgAZg3L2fZYf7Hjj/mf6z/Z42/q+YAyTE1Q60B0QaiNwDleWt/rf1vXYCs57UL4Wj/z+5/7/iL9v9q44/5z/xn/ueZ/wiA4ABED0AWgNwLAPN/Lf8z/5n/M8efWgDUBoK2470LkTQQrQuStABoJ4bV8a0TsTUO3gNxtf57jb8o/0ePP+Y/85/5n3f+IwCiAxA8AFkAci8AzH/mP/M/7/yftgXgPfBGB0C78GgDsNrAkxZArf937X+0/V7/e40/5j/zn/m/n//dxh8LAAsAC8B+/vcaf8x/5j/zfz//O46/OROhdQJ6OyJqIehdCHsnYLT/R/U/2n6v/3cZf8z/Ncc/839v/y8w/lgAWAD27X+0/QMWAOb/Bv5n/jP/B/n/c0Mkw1IDatdrdrXHrfZr5bWOt7bD275kV3vs7f9V+t87/qL9HzX+JDvMf+Y/8//I+c8CEByApQZgq/9X6T8LAAKA+c/8Z/43CgBtg6yB6HWAFa1dbTus7bH2X2rHbHve/h/d/97xF+3/1ccf85/53+N/5v+y4893QLAAxEwAFoC+8Rft/9XHH/Of+d/jf+b/suOvPcgAAACwLeENAAAAgPmENwAAAADmE94AAAAAmM/XP/79V0dZQev9XuV72+/E86mjtZxX+bKeVnun9L+3/d38Y6S3Hm/7rfev0v/uef/8SG95qb7W+1vtt7ZD6w+z/69nE631WMvV7l+l/df3PxAAnQvBngkQAdDX/m6iEyACoHPeIwAs9bj7HwHQ1f5reAKX6h1dXmp/9wLQl0Bnl7e2v2YvS/8l+92MTqBS+dZ6tOVq7Z1lX9v/5vnvk/Ba6xktALzbPUoI/LKjTISzEnhrea9+1uwjALoXgLMTIALgs/1uEABj7Wv73zz/EQAIAATAvuW7F4DYBDTLvvb86QLAndMFQO2+2fYHgQDw6b9bPJwfndfut5b3EiC97f9NdAKOLt+9ACAAEAAdIADm2B8EAgABcLYAGP0IPrp89wKwdgLV1jfL/mr9t9bjvmhlEQCj7GvLDxIEpwsAr3aMFgStiXCWAGgt79WPenkEQOcCgABAAHSAAOizry2PAPhxv7Y8AuD9/bPKxwuAssBqCXwTAVDery3vncBnt3+18qPi2ow2cXklzlMFwOj6quPkbAEQ7RfRzuRH6LMFwHi/RCfg6PK9AxABsFX5UXFtBgHQZ9+rXc3jBAEw0i+iHQRAZ7+iE3B0+d4BuHkC3K39o/rf65dmehNVdAJexb63X9XzP1YASOXL66PsjvKLaL8zce4uACQ7CACpfO8APCQB7tJ+BEBneQQAAmCA3VF+Ee0jAJILAG09zon/94DXYS1Xu99qt1bv6gm813/e9biPHa9EJdXTmih7y9fu9/ZDa/u75/3nRKdNzNZy1vaMar+1Hnf/C4nOmpi9y9XuX6X9CAC3hQABgABoAAHg4wcEwMf6RrXfWo+7/xEAXe2/vAMCAAAAWxDeAAAAAJiPT0W9j4CiHiFBEQf7IyRVOal8enq3sCgfugUYTdgW1iH0+i+6fAdeDkQAnAACIIjoBJi9/OYgAGL9F12+g17H2RK29SUYqTz4EPUSTHqsCas8T/m+8pvT+xJudnr9F13egV4HIgBOAAEQRHQCzF5+cxZIIFsTncAXiF+vAxEAJzFKALAVUEGbwLTnKW8rvzkLJJCt0fpPe352eQd6HYgAOAkEwGSiE2D28puDAJjjPwSA6MjPiVyb2HkZMBZvAVATAtH9XAYpgdUSGeV9yh+ClEhI/D5+sybw0eUd8HIkAuAEEACTiU6A2csfAgJgjt8QAGrH9iVwBEAMvAQ4GW3iKu+nvE/5zbEmfISAr/+iyzswyrEIgB1BAEwmOgFmL785CySQrYlO4AvEb5Rj+xI3iT8GBMBkohNg9vKbs0AC2ZroBL5A/EY5FgGwIwiAyUQnwOzlN2eBBLI10Ql8gfh5OZJH/idgTejaciR+ASmRSYmL8n3lN0dKJCT+sf6LLt+BlwMRACeAAAgiOgFmL785CIBY/0WXb+X/AfoGzIAXoPKXAAAAAElFTkSuQmCC
```

Comment:
Hmm... I'd rather have an image hosted on imgur, that a long `data:` URI in the yaml file. :)